data/data/powervs/master/main.tf

```diff
@@ -0,0 +1,39 @@
+# TODO(mjturek): network and image data blocks can be in main module
+#                as master and bootstrap will be using the same
+#                network and image. Once we add in master module, make
+#                the move.
+data "ibm_pi_network" "network" {
+  pi_network_name      = var.network_name
+  pi_cloud_instance_id = var.cloud_instance_id
+}
+
+data "ibm_pi_image" "master_image" {
+  pi_image_name        = var.image_name
+  pi_cloud_instance_id = var.cloud_instance_id
+}
+
+# Create the bootstrap instance
```

Comment:
copy/paste fix: Create the master instance

Reply:
Done.